packages/workflow/src/Extensions/NumberExtensions.ts

```diff
@@ -86,56 +94,106 @@ function toDateTime(value: number, extraArgs: [DateTimeFormat]) {
 
 ceil.doc = {
 	name: 'ceil',
-	description: 'Rounds up a number to a whole number.',
+	description: 'Rounds the number up to the next whole number',
+	examples: [{ example: '(1.234).ceil()', evaluated: '2' }],
 	returnType: 'number',
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-ceil',
 };
 
 floor.doc = {
 	name: 'floor',
-	description: 'Rounds down a number to a whole number.',
+	description: 'Rounds the number down to the nearest whole number',
+	examples: [{ example: '(1.234).floor()', evaluated: '1' }],
 	returnType: 'number',
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-floor',
 };
 
 isEven.doc = {
 	name: 'isEven',
-	description: 'Returns true if the number is even. Only works on whole numbers.',
+	description:
+		"Returns <code>true</code> if the number is even. Throws an error if the number isn't a whole number.",
+	examples: [
+		{ example: '(33).isEven()', evaluated: 'false' },
+		{ example: '(42).isEven()', evaluated: 'true' },
+	],
 	returnType: 'boolean',
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-isEven',
 };
 
 isOdd.doc = {
 	name: 'isOdd',
-	description: 'Returns true if the number is odd. Only works on whole numbers.',
+	description:
+		"Returns <code>true</code> if the number is odd. Throws an error if the number isn't a whole number.",
+	examples: [
+		{ example: '(33).isOdd()', evaluated: 'true' },
+		{ example: '(42).isOdd()', evaluated: 'false' },
+	],
 	returnType: 'boolean',
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-isOdd',
 };
 
 format.doc = {
 	name: 'format',
 	description:
-		'Returns a formatted string of a number based on the given `LanguageCode` and `FormatOptions`. When no arguments are given, transforms the number in a format like `1.234`.',
+		'Returns a formatted string representing the number. Useful for formatting for a specific language or currency. The same as <a target="_blank" href=”https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat”><code>Intl.NumberFormat()</code></a>.',
+	examples: [
+		{ example: "(123456.789).format('de-DE')", evaluated: '123.456,789' },
+		{
+			example: "(123456.789).format('de-DE', {'style': 'currency', 'currency': 'EUR'})",
```

Comment:
I would not change this, this extension is an exact copy of `Intl.NumberFormat.format`, because of that we can refer to the MDN docs and everything is the same. That would no longer work if we change the API. I agree the API is verbose, but I don't think it's worth changing it right now.